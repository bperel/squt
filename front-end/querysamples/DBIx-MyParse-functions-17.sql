 SELECT ROUND(1,2) 
