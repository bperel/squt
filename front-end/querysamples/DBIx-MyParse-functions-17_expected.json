{
  "Warning": {
    "No alias": {
      "round": ["function", "SELECT"]
    }
  },
  "Functions": {
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "1": "1",
        "2": "2"
      },
      "name": "round",
      "alias": "0"
    }
  }
}